src/main/java/io/appform/dropwizard/actors/base/UnmanagedConsumer.java

```diff
@@ -109,6 +110,12 @@ public void handleDelivery(String consumerTag, Envelope envelope,
                 }
             }
         }
+
+        private MessageProperties messageProperties(final Envelope envelope) {
+            return MessageProperties.builder()
```

Comment:
Instead of builder, use constructor. So that if something else is added tomorrow, this function will need to handle it.

Reply:
added.